go.mod

```diff
@@ -21,11 +21,11 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/mattn/go-shellwords v1.0.12
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/moby/buildkit v0.11.6
+	github.com/moby/buildkit v0.11.5
 	github.com/moby/term v0.0.0-20221205130635-1aeaba878587
 	github.com/morikuni/aec v1.0.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b
+	github.com/opencontainers/image-spec v1.1.0-rc.3
```

Comment:
Please use `v1.1.0-rc3` instead; `v1.1.0-rc.3` ranks lower than `v1.1.0-rc2` according to SemVer (due to the missing dot); https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc3

See https://github.com/moby/moby/pull/45437